static/src/stylesheets/module/commercial/paidfor/_paidfor-popup.scss

```diff
@@ -1,48 +1,27 @@
 .popup--paidfor {
     background: $brightness-7;
     box-shadow: none;
-    left: 100%;
-    top: 100%;
-    padding: 16px;
-    color: #ffffff;
-    transform: translate(-50%, 0);
-    width: 240px;
-    border-radius: 4px;
-    font-weight: 700;
-    z-index: 32000;
+    left: -$gs-gutter / 2;
+    top: $gs-baseline * 4;
+    padding: $gs-baseline $gs-gutter;
+    color: $brightness-100;
+    width: gs-span(3);
+    border-radius: $gs-baseline / 4;
+    z-index: $zindex-main-menu;
```

Comment:
Same as above - some of these classes are also in the amp stylesheets:

https://github.com/guardian/frontend/blob/59e3f4df36230492783d4771a379cea8eba89607/common/app/views/fragments/amp/stylesheets/glabs.scala.html

Do they also need updating there? No worries if fixing paid for AMP is outside of this PR!

Reply:
Thanks Kate, very good point! I've updated AMP to reflect my changes, and tidy it up a bit.